FrontEnd/src/reducers/loginReducer.js

```diff
@@ -60,7 +63,9 @@ export default (state = DEFAULT_STATE, action) => {
             localStorage.setItem('userId', userId);
             localStorage.setItem('role', action.res.rolle);
             localStorage.setItem('request_token', action.res.request_token);
-            return {...state, userId, role: action.res.rolle, loggedIn: true, attemptingLogin: false, request_token: action.res.request_token}
+            return loop({...state, userId, role: action.res.rolle, loggedIn: true, attemptingLogin: false, request_token: action.res.request_token},
```

Comment:
Nur so aus interesse: Was bewirkt dieses loop() in einem return?
Ist das so ein Redux Ding?